src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/ConnectionFactory.scala

```diff
@@ -0,0 +1,116 @@
+package com.microsoft.partnercatalyst.fortis.spark.sinks.cassandra
+
+import java.nio.file.{Files, Path, Paths}
+import java.security.{KeyStore, SecureRandom}
+import javax.net.ssl.{KeyManagerFactory, SSLContext, TrustManagerFactory}
+
+import com.datastax.driver.core._
+import com.datastax.driver.core.policies.{DCAwareRoundRobinPolicy, ExponentialReconnectionPolicy, TokenAwarePolicy}
+import com.datastax.spark.connector.cql.CassandraConnectorConf.CassandraSSLConf
+import com.datastax.spark.connector.cql.{CassandraConnectionFactory, CassandraConnectorConf, MultipleRetryPolicy}
+import org.apache.commons.io.IOUtils
+
+/**
+  * Copy-pasted version of DefaultConnectionFactory to over-write the load balancing policy
+  * There is only one change called out by START/END comments below
```

Comment:
What if we do something like this instead of copy-pasting so much code?

```
class FortisConnectionFactory extends CassandraConnectionFactory {
  override def createCluster(conf: CassandraConnectorConf): Cluster = {
    val lbp = new TokenAwarePolicy(new DCAwareRoundRobinPolicy.Builder().build())
    DefaultConnectionFactory.clusterBuilder(conf).withLoadBalancingPolicy(lbp).build()
  }
}
```

Reply:
Nice one.